src/test/java/com/coherentsolutions/aqa/java/api/makarevich/ZipCodeTest.java

```diff
@@ -0,0 +1,98 @@
+package com.coherentsolutions.aqa.java.api.makarevich;
+
+import com.coherentsolutions.aqa.java.api.makarevich.httpClient.HttpResponseWrapper;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import static com.coherentsolutions.aqa.java.api.makarevich.configuration.Configuration.API_ZIPCODES_ENDPOINT;
+import static com.coherentsolutions.aqa.java.api.makarevich.configuration.Configuration.API_ZIPCODES_EXPAND_ENDPOINT;
+import static com.coherentsolutions.aqa.java.api.makarevich.constants.Constants.STATUS_CODE_200_OK;
+import static com.coherentsolutions.aqa.java.api.makarevich.constants.Constants.STATUS_CODE_201_CREATED;
+import static java.lang.System.currentTimeMillis;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class ZipCodeTest extends TestBase {
+    /*
+    Scenario #1
+    Given I am authorized user
+    When I send GET request to /zip-codes endpoint
+    Then I get 200 response code
+    And I get all available zip codes in the application for now
+    */
+
+    @Test
+    public void testGetZipCodes() throws IOException {
+        HttpResponseWrapper response = httpMethods.get(API_ZIPCODES_ENDPOINT);
+        ArrayList<String> arrayList = response.getReponseBodyAsArray();
+        assertEquals(STATUS_CODE_200_OK, response.getStatusCode(), "Expected status code to be 200");
+        assertFalse(arrayList.isEmpty(), "List of zip codes is empty");
+    }
+
+    /*
+    Scenario #2
+    Given I am authorized user
+    When I send POST request to /zip-codes/expand endpoint
+    And Request body contains list of zip codes
+    Then I get 201 response code
+    And Zip codes from request body are added to available zip codes of application
+    */
+
+    @Test
+    public void testAddZipCodes() throws IOException {
+        String zipcode = String.valueOf(currentTimeMillis());
+        HttpResponseWrapper response = httpMethods.post(API_ZIPCODES_EXPAND_ENDPOINT, "[\"" + zipcode + "\"]");
```

Comment:
"[\"" + zipcode + "\"]" - I don't like that construction. List.of() method should help you to create array in a simple way

Reply:
Implemented the addZipCode(String... zipCode) method with List.of() to be able to add multiple String values as an array